sale_payment_method/sale.py

```diff
@@ -305,6 +305,11 @@ def action_view_payments(self, cr, uid, ids, context=None):
         return action
 
     def action_cancel(self, cr, uid, ids, context=None):
+        if context is None:
+            context = {}
+        if context.get('force_cancel', False):
```

Comment:
`, False` is useless here as the return value of `{}.get()` is `None` when the key does not exist.